coderkevin/fresh-data
# Add redux to hello-world example

This adds redux to the hello-world example in preparation of the next step, which is using fresh-data in the example here.

To Test:
1. `cd examples/hello-world`
2. `npm start`
3. Visit `http://localhost:3000/` and ensure page loads and says "Hello World!"
4. Observe redux dev tools to ensure "World" is in state.
